integration/spark/src/main/scala/org/apache/spark/sql/optimizer/CarbonFilters.scala

```diff
@@ -152,13 +152,25 @@ object CarbonFilters {
     }
 
     def getCarbonExpression(name: String) = {
```

Comment:
in 'createFilter' method,  convert CarbonArrayContains filter to ArrayContainsExpression

Reply:
I want reuse existing equalsTo code, I don't see any advantage of making new expression